tests/test_runner.sh

```diff
@@ -19,4 +19,4 @@ function execute() {
 TESTS=${@:-.}
 
 # Run the tests.
-execute time bats --tap $TESTS
+execute time bats -j 4 --tap $TESTS
```

Comment:
I would use `$(nproc)` here instead of hard coding any count, should make it much faster when run locally.